.github/workflows/new-release-tag-docker.yml

```diff
@@ -35,6 +35,17 @@ jobs:
         run: |
           git tag -a ${{ github.event.inputs.release_number }} -m "[bot] release ${{ github.event.inputs.release_number }}"
           git push origin ${{ github.event.inputs.release_number }}
+      - name: Install Helm
```

Comment:
Could you please replace that stanza with the helm action, such as

```
      - name: Install Helm
        uses: azure/setup-helm@v1
        with:
          version: v3.4.0
```

for now?